README.md

```
@@ -37,12 +37,59 @@ This repository contains a sleek, reactive frontend, but it leans on a robust ba
 
 ---
 
+## Frontend Setup
+
+### Minimal config
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)